l10n_it_intrastat_statement/readme/USAGE.rst

```diff
@@ -0,0 +1,60 @@
+
+N.B.: "Operazioni contabili" è visibile solo se vengono abilitate le funzionalità contabili complete.
+
+Nella parte superiore della maschera, inserire i dati:
+
+- Azienda: popolato in automatico con il nome dell'azienda;
+- Partita IVA contribuente: la partita IVA, popolata in automatico con il nome dell'azienda;
+- Data di presentazione: popolata in automatico con la data corrente;
+- Anno: l'anno di presentazione, scelto dal menù a tendina che visualizza gli anni fiscali configurati a sistema;
+- Tipo periodo: l’orizzonte temporale a cui fa riferimento la dichiarazione, scelto da menù a tendina con le voci “Mese” o “Trimestre”;
+- Periodo: il periodo temporale a cui fa riferimento la dichiarazione. Inserire il numero del mese (es. 9 per settembre, se nel campo "Tipo periodo" è stato selezionato “Mese”, oppure in numero del trimestre (es: 1 per il trimestre gennaio-marzo), se nel campo "Tipo periodo" è stato selezionato “Trimestre”;
+- Caselle di selezione “Cessioni” e “Acquisti”: da selezionare in base alla tipologia di operazioni che si vogliono inserire nella dichiarazione;
+- Numero: progressivo della dichiarazione proposto in automatico dal sistema;
+- Tipo di contenuto: selezionare la voce di competenza dal menù a tendina;
+- Casi speciali: selezionare la voce di competenza dal menù a tendina;
+- Sezione doganale: selezionare la voce di riferimento dal menù a tendina.
+
+.. figure:: https://raw.githubusercontent.com/OCA/l10n-italy/11.0/l10n_it_intrastat/static/img/dichiarazione.png
+   :alt: Dichiarazione
+   :width: 600 px
+
+
+Inseriti e salvati i dati, utilizzare il pulsante «Ricalcola» per popolare la dichiarazione. Per ciascuna scheda (”Cessioni” e “Acquisti”) verranno inserite nelle sezioni di riferimento:
+
+- Cessioni:
+  - Cessione beni - Sezione 1 → fatture di vendita di merci
+  - Rettifica beni - Sezione 2 → note di credito su vendita merci
+  - Cessione servizi - Sezione 3 → fatture di vendita di servizi
+  - Rettifica servizi - Sezione 4 → note di credito su vendita servizi
+- Acquisti:
+  - Acquisto beni - Sezione 1 → fatture di acquisto di merci
+  - Rettifica beni - Sezione 2 → note di credito su acquisto merci
+  - Acquisto servizi - Sezione 3 → fatture di acquisto di servizi
+  - Rettifica servizi - Sezione 4 → note di credito su acquisto servizi
+
+I dati presi dalle fatture e dalle note di credito indicate come soggette ad Intrastat, relative al periodo di riferimento.
+
+N.B.: i record presenti nelle schede "Rettifica beni - Sezione 2" e "Rettifica servizi - Sezione 4", sia per gli acquisti che per le vendite, vanno modificati per inserire i dati obbligatori mancanti.
+
+Inseriti i dati e salvata la dichiarazione, è possibile procedere all’elaborazione dei file da inviare all’Agenzia delle Dogane tramite l’apposito pulsante «Esporta file». 
+
+.. figure:: https://raw.githubusercontent.com/OCA/l10n-italy/11.0/l10n_it_intrastat/static/img/export_file.png
+   :alt: Export file
```

Comment:
Anche qui come sopra manca l'immagine.

Lo stesso problema è presente nella 12.0, vedi "P.S." in https://github.com/OCA/l10n-italy/pull/1546#issue-347770769

Reply:
> Anche qui come sopra manca l'immagine.
> 
> Lo stesso problema è presente nella 12.0, vedi "P.S." in [#1546 (comment)](https://github.com/OCA/l10n-italy/pull/1546#issue-347770769)

@primes2h non ho idea di che immagini siano, secondo me si possono togliere

Reply:
@

> > Anche qui come sopra manca l'immagine.
> > Lo stesso problema è presente nella 12.0, vedi "P.S." in [#1546 (comment)](https://github.com/OCA/l10n-italy/pull/1546#issue-347770769)
> 
> @primes2h non ho idea di che immagini siano, secondo me si possono togliere

@glauco70 ok, per la 12.0 fatto in https://github.com/OCA/l10n-italy/compare/5fb252e..76f31b6#diff-9078fca374979bd67e959f7a8e84d8c9